### DISCUSSION TITLE
any way to disable ui or frontend?

### DISCUSSION THREAD

**Discussion by haidarabdillah** [blockscout/blockscout#8625] [General]
so i just want to run with my own custom ui

**Comment by ekawibisono** [comment]
hello @haidarabdillah 
How do you run blockscout? Do you use manual method or docker-compose?

**Comment by haidarabdillah** [comment]
have tried this docker 
https://github.com/blockscout/blockscout/blob/master/docker-compose/docker-compose-no-build-external-frontend.yml

use env :
    environment:
      ETHEREUM_JSONRPC_VARIANT: 'geth'
      BLOCK_TRANSFORMER: 'clique'

running backend but the cpu usage very aggresive 
![image](https://github.com/blockscout/blockscout/assets/113880800/371238f4-6af7-4cba-aeb6-f6b362bf9335)

**Reply by ekawibisono** [reply]
Mention your server or machine specifications

**Reply by haidarabdillah** [reply]
im running on my local laptop using 8core 16thread ryxen 7 5700u and ram 32gb running on ssd pcie 3, good internet connection

**Reply by ekawibisono** [reply]
Please try to check again on your application that is running
